spec/models/conversation_spec.rb

```diff
@@ -0,0 +1,52 @@
+require 'rails_helper'
+
+describe Conversation, type: :model do
+  let!(:user)            { create(:user) }
+  let!(:second_user)     { create(:user) }
+
+  subject { build(:conversation) }
+
+  describe 'validations' do
+    it { is_expected.to have_many(:conversation_users) }
+    it { is_expected.to have_many(:users).through(:conversation_users) }
+    it { is_expected.to have_many(:messages) }
+  end
+
+  describe '.create_conversations' do
+    let!(:conversation) { create(:conversation) }
+
+    subject(:create_conversations) do
+      Conversation.create_conversations([user, second_user])
+    end
+
+    it 'creates a conversation' do
+      expect { subject }.to change { Conversation.count }.by(1)
+    end
+
+    it 'includes each user' do
```

Comment:
You could check the same for `second_user`, just to check that both users were included in the conversation.